src/modules/preferences.js

```diff
@@ -100,7 +101,8 @@ PassFF.Preferences = (function () {
     'urlFieldNames'       : ',',
     'otpauthFieldNames'   : ',',
     'autoFillBlacklist'   : ',',
-    'recognisedSuffixes'  : ','
+    'recognisedSuffixes'  : ',',
+    'filterFieldRegex'    : ',',
```

Comment:
This could potentially be error prone. Since `|` acts as a natural separator in regex, essentially all this does is replace `,` with `|` in the final result. And, as unlikely as it may seem, someone may use a `,` in their regex, which will cause a bug. Is it possible to make this field *not* a list, and just input a raw regex?

Reply:
You're right; I didn't take that into consideration. I'll fix this in my next push, addressing this and your next comment.

Then, I'd like to add a note about how the regex needs to be separated by pipes for each different match. Do you have any ideas on where this should go?

Under my addition to the README as a note?
Under the actual entry in the preferences? And if you think that is preferable, what would this look like? I'm completely unfamiliar with writing HTML.

Reply:
This might be able to be done with just the default field value and the label.

The default field value will be something like `notes|attributes|attachments` which should answer the question of "How do I target multiple items."

And the label can make it clear that it is a single regex which tests against the full path of the entry. Something like `Single regex for hiding items (tests against the full path):`

Reply:
A fairly simple solution. Guess I'll just have to hang out in the issues to make sure it's clear to users. Thanks for your input! Push should show up soon. Hopefully everything has been addressed.

Reply:
Having only a single regular expression for all kinds of tests is a bit tedious in case one of the tests is supposed to test from the beginning of the path, i.e., using `^` (or, same problem: `$`). Maybe we could instead have a multiline textfield and have one test per line?

Reply:
@tuxor1337 Before I push, is this what you had in mind? https://gist.github.com/cole-h/1cdacc1d89a722869d405e80c41ade21 (diff between current branch HEAD)

Reply:
Yes, that looks okay. Just one remark: the `<textarea>` isn't a *void element* according to the HTML5 specification (https://w3c.github.io/html/syntax.html#void-elements). So it can't be self-closed using a slash `/` in the start tag but it has to be followed by the corresponding end tag `</textarea>`. Please change this before pushing.

Reply:
Whoops. Thanks for catching that. As you can tell, I don't use HTML much.

Reply:
I have nothing to add other than saying the textarea is a good compromise, the implementation looks good, and good on you for making this PR out of your comfort zone.